scheduler/reconcile.go

```diff
@@ -218,23 +218,22 @@ func (a *allocReconciler) Compute() *reconcileResults {
 
 	// Mark the deployment as complete if possible
 	if a.deployment != nil && complete {
-
-		var status string
-		var desc string
-
 		if a.job.IsMultiregion() {
-			status = structs.DeploymentStatusBlocked
-			desc = structs.DeploymentStatusDescriptionBlocked
+			if a.deployment.Status != structs.DeploymentStatusUnblocking &&
+				a.deployment.Status != structs.DeploymentStatusSuccessful {
```

Comment:
Maybe a small comment on the significance of these particular statues.  Also, should it filter on failed?

Reply:
If it's failed it won't be `complete`. I'll add the comment though.